proposals/csharp-9.0/native-integers.md

```diff
@@ -92,25 +97,26 @@ The tables below cover the conversions between special types.
 | `nint` | `int` | ExplicitNumeric | `conv.i4` / `conv.ovf.i4` |
 | `nint` | `uint` | ExplicitNumeric | `conv.u4` / `conv.ovf.u4` |
 | `nint` | `long` | ImplicitNumeric | `conv.i8` / `conv.ovf.i8` |
```

Comment:
`nint to long` cannot overflow. It is either a sign-extension (on 32-bit) or the same size (on 64-bit).

Unless the language is trying to future proof for some .NET where pointers could be larger than 64-bit (in which case the entire BCL and more would need to update to handle this stuff...), then I think it can just be `conv.i8`

Reply:
It looks like the table is incorrect. The implementation uses `conv.i8` even in a `checked` context.

Reply:
Will fix the table.
The implementation indeed uses `conv.i8` for both unchecked and checked:
`            conversions(sourceType: "nint", destType: "long", ImplicitNumeric, expectedImplicitIL: conv("conv.i8"), expectedExplicitIL: conv("conv.i8"));`